common/src/main/java/io/honeycomb/opentelemetry/EnvironmentConfiguration.java

```diff
@@ -174,6 +187,12 @@ public static void enableOTLPTraces() {
         final String endpoint = getHoneycombTracesApiEndpoint();
         final String apiKey = getHoneycombTracesApiKey();
         final String dataset = getHoneycombTracesDataset();
+        final String serviceName = getServiceName();
+
+        // helpful to know if service name is missing
+        if (!isPresent(serviceName)) {
+            System.out.printf("WARN: %s%n", getErrorMessage("service name", SERVICE_NAME));
```

Comment:
We'll need to add our additional info about why to set service name here. Maybe we have a different way to declare the warning message strings once now that we want to say different things in the warnings about different settings?